consensus/enclave/impl/src/lib.rs

```diff
@@ -205,6 +207,15 @@ impl ConsensusEnclave for SgxConsensusEnclave {
         Ok(self.ake.get_identity().get_public_key())
     }
 
+    fn get_fee_recipient(&self) -> Result<FeePublicKey> {
```

Comment:
Is it intentional to both return `Result` and `unwrap()` the errors?

Reply:
Good point - fixed